frontend/packages/dev-console/integration-tests/features/Helm/helm-compatability-ODC-3430.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Helm Chart
+    User will be able to update the chart versions or values config of a helm release
+
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke, @manual
+Scenario: Compatible helm charts
+    Given user is at +Add page
+    When user clicks on the Helm Chart card on the +Add page
+    Then user sees the helm charts compatible with the kubernetes version of the cluster
+
+
+@regression, @smoke, @manual
+Scenario: Check the meta data for the importing helm charts from index.yaml
+    Given user is at +Add page
```

Comment:
```suggestion
    Given user is on the +Add page
```

Reply:
The same statement used in other feature files so that we can reuse it later while writing the step definition.